src/reference-designs/tap-architecture-planning.md

```diff
@@ -0,0 +1,218 @@
+# Tanzu Application Platform Planning and Architecture Reference
+## Architecture Overview 
+The goal of this reference document is to help provide a standard architecture for deploying Tanzu Application Platform (TAP). This reference will cover topics such as kubernetes requirements and cluster layout for Tanzu Application Platform. For guidance on how to deploy your particular kubernetes distribution you should refer to the reference architecture for that distribution.
+This architecture should give you an path to creating a production deployment of Tanzu Application Platform, however you should not feel constrained exactly this path if your specific use cases lead you to architectural differences.
+
+### Cluster Layout
+For production deployments, we recommend that there be two fully independent instances of TAP. One instance will be simply for operators to conduct their own reliability tests and the other that will host development, test, qa, and production environments isolated by separate clusters.
+![](img/tap-architecture-planning/overview.png)
+| Decision ID   | Design Decision   | Justification | Implication
+|---            |---                |---            |---
+|TAP-001  | Install using multiple clusters         |  Utilizing multiple clusters allows you to separate your workloads and environments while still leveraging combined build infrastructure   |  Multiple cluster design requires more installation effort and possibly maintenance versus a single cluster design
```

Comment:
I am not against, this but this is a departure from other docs we have written. Should we open Issues to get the same decisioning content for other docs?

Reply:
I think we should aspire to add that detail to all reference architectures, I don't know how that should be prioritized.

Reply:
Let' talk about this on the next call